### DISCUSSION TITLE
WPA3 ?

### DISCUSSION THREAD

**Discussion by raccoon0815** [tbnobody/OpenDTU#416] [Q&A] [answered]
Hat jemand openDTU mit WPA3 laufen?

**Comment by tbnobody** [comment]
Laut Internetz sollte das ohne Anpassungen am Code klappen. (Kann es aber mangels Hardware nicht prüfen)

**Reply by raccoon0815** [reply]
Ok bekomme am Montag eine Fritz Box zum Testen - melde mich dann

**Comment by Solarteur** [comment, marked as answer]
OpenDTU lief bei mir die ganze Zeit auf WPA3, ohne dass ich es wusste ;-)
(FritzBox 7530, FRITZ!OS: 7.39-101552 BETA, WPA-Modus: WPA2 + WPA3

![ESP32_WPA3](https://user-images.githubusercontent.com/117977338/206637108-6402e716-c43a-4763-86fe-bc01ced7321d.png)
